zubhub_frontend/zubhub/src/views/my_activities/MyActivities.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { connect, useSelector } from 'react-redux';
+import {
+  getActivities,
+  getMyActivities,
+  activityToggleSave,
+  setActivity,
+  getUnPublishedActivities,
+} from '../../store/actions/activityActions';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import Activity from '../../components/activity/activity';
+import styles from '../../assets/js/styles/views/activities/activitiesStyles';
+import { makeStyles } from '@material-ui/core/styles';
+import ErrorPage from '../error/ErrorPage';
+import DefaultStyles from '../../assets/js/styles';
+import { Grid, Box, Typography } from '@material-ui/core';
+import LoadingPage from '../loading/LoadingPage';
+const useStyles = makeStyles(styles);
+
+function MyActivities(props) {
```

Comment:
This get's loaded at `activities/my-activities` and generally shows user specific published activities. 
It will make sense if it can show all, and then add tags for the `unpublished`